src/mirror.c

```diff
@@ -121,7 +121,6 @@ static int unset_mirror_url()
 	free_string(&content_url);
 	set_default_version_url();
 	set_default_content_url();
-	get_latest_version("");
```

Comment:
the call to get_latest_version("") is necessary to reload the defaults in the mirror, after unsetting it... so I do think this belongs to the unset_mirror_url() function.

It is true that at the moment handle_mirror_if_stale() is the only function that really continues doing more stuff after unsetting the mirror, but what if in the future we need to unset the mirror as part of other process, we will have to reload the defaults again.

Reply:
yes. I didn't have a better idea for now. I'd be more concerned if this function was exported, but as it's a local function we have more control on where it's used.